jobs/async-upload/tests/integration/test_integration_async_upload.py

```
@@ -0,0 +1,365 @@
+    """Test the complete async-upload job integration.
+    
+    This test:
+    1. Creates a RegisteredModel, ModelVersion, and placeholder ModelArtifact
+    2. Downloads and uploads an ONNX model to MinIO
+    3. Creates and applies a Kubernetes job using kustomize
+    4. Waits for job completion
+    5. Validates the final artifact state
+    """
+    
+    # Configuration
+    mr_host_url = os.environ.get("MR_HOST_URL", "http://localhost:8080")
```

Ruff (0.12.2)

Local variable `mr_host_url` is assigned to but never used

Remove assignment to unused variable `mr_host_url`

(F841)